### ISSUE TITLE
Feature request: allow hiding inactive channels

### ISSUE THREAD

**Issue by Mikaela** [communi/communi-sailfish#98]
WeeChat's `buffers.pl` script (adds channel sidebar) allows hiding channels from it if they have been inactive for time specified in config. Mobile IRC clients could really use this feature as the channel lists can be very large if used with bouncer like ZNC.

At the moment I have 7/83 channels visible in WeeChat.

**Comment by jpnurmi** [comment]
Have you noticed that the search button in the left side panel can be toggled between all-activity-highlighted states? It's not exactly the same, but maybe helps a bit...

**Comment by Mikaela** [comment]
No, I didn't and I am using your \*playback module as workaround and it works enough well.
